test/go/genmock.sh

```diff
@@ -4,7 +4,6 @@ set -e
 
 export GOPATH=$(mktemp -d -t gopath-XXXXXXXXXX)
 
-# TODO: Once we dropped support to go 1.15, add "@v1.5.0" suffix to go install
```

Comment:
NOTE: I removed this TODO because I realized that without "@v1.5.0", `go install` will actually pick the version according to `go.mod`, which is better because we don't need to keep the two versions in sync.